src/main/resources/openapi.json

```diff
@@ -753,6 +753,74 @@
                 }
             ]
         },
+        "/admin/topics/{topicname}": {
+            "post": {
+                "tags": [
+                    "Topics"
+                ],
+                "description": "Creates a topic with given topic name.",
+                "operationId": "createTopic",
+                "responses": {
+                    "200": {
+                        "description": "Topic metadata",
+                        "content": {
+                            "application/vnd.kafka.v2+json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/TopicMetadata"
```

Comment:
Is topic creation really returning anything? I see the API just having void as well as your implementation.